kitsune/questions/api.py

```diff
@@ -386,6 +388,18 @@ def auto_tag(self, request, pk=None):
         question.auto_tag()
         return Response(status=status.HTTP_204_NO_CONTENT)
 
+    @action(methods=['POST'], permission_classes=[permissions.IsAuthenticated])
```

Comment:
`permission_classes=[permissions.IsAuthenticated]` will allow any authenticated user to attach an image to a question. You probably want to leave this off and use the default permissions for this class, which are `permission_classes = [OnlyCreatorEdits, permissions.IsAuthenticatedOrReadOnly]`.